frontend/src/lib/utils.tsx

```diff
@@ -1447,3 +1447,18 @@ export function processCohort(cohort: CohortType): CohortType {
         },
     }
 }
+
+export function isVersionLT(version1: string, version2: string): boolean {
```

Comment:
nit: I'd be in favor of the explicit name rather than the acronym even as well-established as this is